state-transition/core/state_processor_staking_test.go

```diff
@@ -336,15 +316,10 @@ func TestTransitionCreateValidator(t *testing.T) {
 		t,
 		st,
 		&types.BeaconBlockBody{
-			ExecutionPayload: &types.ExecutionPayload{
-				Timestamp:    blk1.Body.ExecutionPayload.Timestamp + 1,
```

Comment:
we used blk1 here but this is wrong. The timestamp is hardly verified here, since we mocked the interaction with the EVM, but use blk.Timestamp is the correct thing to do (we increase of 1 second the parent payload timestamp)